Description: Fix test failure with groff 1.21
 This has been fixed more correctly upstream, but the fix is intrusive to
 backport.
Author: Colin Watson <cjwatson@debian.org>
Forwarded: not-needed
Last-Update: 2011-02-08

Index: b/man/check-man
===================================================================
--- a/man/check-man
+++ b/man/check-man
@@ -37,7 +37,7 @@
 # pages should probably be rewritten without tbl.
 case $LINGUA in
 	de|id|it|nl|pl|ru)
-		errors="$(echo "$errors" | $EGREP -v 'table wider than line width')" || true
+		errors="$(echo "$errors" | $EGREP -v '^warning:.*around line [0-9]*:$|table wider than line width')" || true
 		;;
 esac
 
